python/ctsm/crop_calendars/interpolate_gdds.py

```diff
@@ -0,0 +1,154 @@
+"""
+Interpolate a maturity requirement (GDD) file
+"""
+
+import os
+import sys
+import argparse
+import logging
+import xarray as xr
+
+# -- add python/ctsm  to path (needed if we want to run this stand-alone)
+_CTSM_PYTHON = os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir)
+sys.path.insert(1, _CTSM_PYTHON)
+
```

Comment:
Should this script have some python testing for it?

Reply:
If it should, that could be done by opening an issue and doing that later.

Reply:
Sure; issue added as #2602.